main.js

```diff
@@ -0,0 +1,420 @@
+const startButton = document.getElementById('start-btn');
+startButton.addEventListener('click', startQuiz);
+
+let currentQuestionNumber = 0;
+
+// Questions
+
+const questions = [
+  {
+    question: 'Who invented JavaScript?',
+    imgURL: 'https://media.giphy.com/media/W7dBXzbnEpOBG/source.gif',
+    answers: [
+      { name: 'Douglas Crockford' },
+      { name: 'Sheryl Sandberg' },
+      { name: 'Brendan Eich', correct: true },
+      { name: 'Random name' }
+    ],
+    topic: 'Introduction JS'
+  },
+  {
+    question: 'Which one of these is a JavaScript package manager?',
+    answers: [
+      { name: 'Node.js' },
+      { name: 'TypeScript' },
+      { name: 'npm', correct: true },
+      { name: 'Random name' }
+    ],
+    topic: 'Introduction JS'
+  },
+  {
+    question: 'Which tool can you use to ensure code quality??',
+    answers: [
+      { name: 'Angular' },
+      { name: 'jQuery' },
+      { name: 'RequireJS' },
+      { name: 'ESLint', correct: true }
+    ],
+    topic: 'Introduction JS'
+  },
+  {
+    question: 'Which tool can you use to ensure code quality??',
+    imgURL: 'https://media.giphy.com/media/W7dBXzbnEpOBG/source.gif',
+    answers: [
+      { name: 'Angular' },
+      { name: 'jQuery' },
+      { name: 'RequireJS' },
+      { name: 'ESLint', correct: true }
+    ],
+    topic: 'Introduction JS'
+  }
+];
+
+// Counter of questions "Progress"
+function showProgress() {
+  let element = document.getElementById('progress');
+  element.innerHTML =
+    'Question ' + (currentQuestionNumber + 1) + ' of ' + questions.length;
+}
+
+// Save the answer & have an array with the chosen answers
+
+let chosenAnswers = [];
+
+function saveTheAnswer(currentQuestionNumber, choiceIndex) {
+  
+  let choices = questions[currentQuestionNumber].answers;
+  // das ist die angeklickte Antwort mit dem Fragenindex und der Antwort
+  console.log(currentQuestionNumber, choices[choiceIndex]);
+  chosenAnswers[currentQuestionNumber] = choices[choiceIndex];
+
+  if (choices[choiceIndex].correct === true) {
+    // alert(`congratulations`);
+    pointsCounter++;
```

Comment:
somehow I think you could calculate the total points at the end when the quiz is over, right ? (because you are not showing the points while the player is playing) .. and the good thing is then that you don't have to take care to substract points when the user goes "back"